public/tokenlist.json

```
@@ -0,0 +1,8611 @@
+{
+  "name": "Uniswap Labs Default",
```

open to other names, let's just clarify this is now a custom list though. Similarly we should change the version to 1.0.0, currently it's 7.0.0
```suggestion
  "name": "Railgun Shielder",
```

---

Cool, and I will add a description field with the versions of the lists we combined